src/dmd/backend/obj.d

```diff
@@ -198,6 +198,101 @@ version (Windows)
     int  MsCoffObj_seg_debugS_comdat(Symbol *sfunc);
 }
 
+version (Posix)
```

Comment:
Shouldn't this be `static if (ELFOBJ || MACHOBJ)`?

Reply:
It'll change anyway when the D conversion gets further along.

Reply:
OK.